unison-cli/package.yaml

```diff
@@ -117,7 +117,7 @@ executables:
         other-modules: Paths_unison_cli
     source-dirs: unison
     main: Main.hs
-    ghc-options: -threaded -rtsopts -with-rtsopts=-I0 -optP-Wno-nonportable-include-path
+    ghc-options: -threaded -rtsopts "-with-rtsopts=-I0 -T" -optP-Wno-nonportable-include-path
```

Comment:
I don't have any information on it, but I'm guessing that collecting and maintaining run time stats isn't completely free, does it make sense to enable this for our production executables? Should we instead find a way just to enable it for our transcript executable?

Reply:
It's very cheap & normal to have on in production for monitoring stats